Makefile

```diff
@@ -172,7 +172,7 @@ lint: agentlint
 # more without -race for packages that have known race detection issues.
 test:
 	$(GO_ENV) go test $(GO_FLAGS) -race ./...
-	$(GO_ENV) go test $(GO_FLAGS) ./pkg/integrations/node_exporter ./pkg/logs ./pkg/operator ./pkg/util/k8s ./component/otelcol/processor/tail_sampling ./component/loki/source/file
+	$(GO_ENV) go test $(GO_FLAGS) ./pkg/integrations/node_exporter ./pkg/logs ./pkg/operator ./pkg/util/k8s ./component/otelcol/processor/tail_sampling ./component/loki/source/file ./component/otelcol/processor/probabilistic_sampler
```

Comment:
Is this change necessary? Does have probabilistic_sampler have race issues?

Reply:
I thought it could be a race issue because tests failed a few times
Removed this change.
Tests are passing now. 
There is one test that failed a few times:
```
--- FAIL: Test_TopicDiscovery (1.00s)
    target_syncer_test.go:60: 
        	Error Trace:	/drone/src/component/loki/source/internal/kafkatarget/target_syncer_test.go:60
        	Error:      	Condition never satisfied
        	Test:       	Test_TopicDiscovery
        	Messages:   	expected topics: [topic1], got: []
```
I'm not sure if it is a flaky test or if it started to fail from time to time because of probabilistic_sampler...
https://drone.grafana.net/grafana/agent/12594/5/2